infrastructure/zk/src/server.ts

```diff
@@ -3,41 +3,61 @@ import * as utils from './utils';
 import * as env from './env';
 import fs from 'fs';
 import * as db from './db/db';
+import * as deploy from './deploy';
 
 import { ethers } from 'ethers';
 
-export async function core() {
+export async function core(docker: boolean) {
     prepareForcedExitRequestAccount();
 
-    await utils.spawn(
-        'cargo run --bin zksync_server --release -- --components=eth-sender,witness-generator,forced-exit,prometheus,core,rejected-task-cleaner,fetchers,prometheus-periodic-metrics'
-    );
+    if (docker) {
+        await deploy.dockerUp('server-core');
+    } else {
+        await utils.spawn(
+            'cargo run --bin zksync_server --release -- --components=eth-sender,witness-generator,forced-exit,prometheus,core,rejected-task-cleaner,fetchers,prometheus-periodic-metrics'
+        );
+    }
 }
 
-export async function web3Node() {
-    await utils.spawn('cargo run --bin zksync_server --release -- --components=web3-api');
+export async function web3Node(docker: boolean) {
```

Comment:
I'd specify false as default value, so we don't need to change all the previous usages

Reply:
updated